rero/rero-ils
# user: enable user profile

* Enable invenio-userprofiles.
* Adds missing templates such as breadcrumbs.
* Removes personal data in the patron profile.
* Adds a RERO ID profile menu.
* Enables invenio-admin.

Signed-off-by: Johnny Mariéthoz <Johnny.Mariethoz@rero.ch>

## Why are you opening this PR?

- Implements: https://tree.taiga.io/project/rero21-reroils/us/1590

## How to test?

- Bootstrap and setup.
- Login with a patron, and you should use two menus. Both should works.
- Login with the admin user and go to https://localhost:5000/admin. This should work.

## Code review check list
